smart_card_connector_app/src/pcsc-api-jstocxxtest.js

```diff
@@ -64,5 +81,33 @@ goog.exportSymbol('testPcscApi', {
         'SmartCardConnectorApplicationTestHelper', {});
     await pcscReadinessTracker.promise;
   },
+
+  // Test `SCardEstablishContext()`.
+  'testScardEstablishContext': async function() {
+    await testController.setUpCppHelper(
+        'SmartCardConnectorApplicationTestHelper', {});
+
+    const apiMessageChannelPair = new GSC.MessageChannelPair();
+    const clientHandler = new ClientHandler(
+        testController.executableModule.getMessageChannel(),
+        pcscReadinessTracker.promise, apiMessageChannelPair.getFirst(),
+        /*clientOrigin=*/ undefined);
+    const api = new GSC.PcscLiteClient.API(apiMessageChannelPair.getSecond());
+
+    const result = await api.SCardEstablishContext(
+        GSC.PcscLiteClient.API.SCARD_SCOPE_SYSTEM);
+    let sCardContext;
+    result.get(
+        (context) => {
+          sCardContext = context;
+        },
+        (errorCode) => {
+          fail(`Unexpected error ${errorCode}`);
+        });
+    assert(Number.isInteger(sCardContext));
```

Comment:
maybe check if it's equal to SUCCESS?

Reply:
Hmm this is a random handle, not an error code. We don't know the exact value to expect here.